v2.0/paths/tasks__delete_tasks_id.yml

```diff
@@ -18,3 +18,6 @@ responses:
   204:
     description: |-
       An empty response will be returned when the task was successfully deleted.
+  404: 
```

Comment:
Does this response also include a body? 

Can we make the description match the format “Returns an error if…” as per https://github.com/box/box-openapi/blob/master/v2.0/paths/chunked_uploads__post_files_id_upload_sessions.yml#L51?